flixel/FlxObject.hx

```diff
@@ -826,6 +826,11 @@ class FlxObject extends FlxBasic
 		return point.subtract(Camera.scroll.x * scrollFactor.x, Camera.scroll.y * scrollFactor.y);
 	}
 	
+	public inline function getPosition():FlxPoint
```

Comment:
Hm.. I was never a huge fan of that, but then again, it might make sense purely for consistency..